AutomatedTesting/Gem/PythonTests/scripting/NodePalette_HappyPath_ClearSelection.py

```diff
@@ -23,56 +21,45 @@ def NodePalette_HappyPath_ClearSelection():
 
     Test Steps:
-     1) Open Script Canvas window (Tools > Script Canvas)
-     2) Get the SC window object
-     3) Open Node Manager if not opened already
-     4) Set some string in the Search box
-     5) Verify if the test string is set
-     6) Clear search string and verify if it is cleared
+     1) Initialize our O3DE editor object
+     2) Open and initialize Script Canvas window (Tools > Script Canvas)
+     3) Get a local handle on the Node Palette QtPy object
+     4) Search for a node (this test uses a node from the Math category)
+     5) Verify the node was found
+     6) Clear search strin
 
     Note:
-     - This test file must be called from the Open 3D Engine Editor command terminal
      - Any passed and failed tests are written to the Editor.log file.
             Parsing the file or running a log_monitor are required to observe the test results.
 
     :return: None
     """
-    from PySide2 import QtWidgets
-
-    import pyside_utils
-    from utils import TestHelper as helper
 
+    #Preconditions
+    from editor_python_test_tools.utils import Report
     import azlmbr.legacy.general as general
+    from editor_python_test_tools.QtPyO3DEEditor import QtPyO3DEEditor
+    from consts.scripting import (NODE_STRING_TO_NUMBER)
 
-    TEST_STRING = "Test String"
-
-    # 1) Open Script Canvas window (Tools > Script Canvas)
     general.idle_enable(True)
-    general.open_pane("Script Canvas")
-    helper.wait_for_condition(lambda: general.is_pane_visible("Script Canvas"), 3.0)
-
-    # 2) Get the SC window object
-    editor_window = pyside_utils.get_editor_main_window()
-    sc = editor_window.findChild(QtWidgets.QDockWidget, "Script Canvas")
-
-    # 3) Open Node Manager if not opened already
-    if sc.findChild(QtWidgets.QDockWidget, "NodePalette") is None:
-        action = pyside_utils.find_child_by_pattern(sc, {"text": "Node Palette", "type": QtWidgets.QAction})
-        action.trigger()
-    node_palette = sc.findChild(QtWidgets.QDockWidget, "NodePalette")
-    search_frame = node_palette.findChild(QtWidgets.QFrame, "searchFrame")
-
-    # 4) Set some string in the Search box
-    search_box = search_frame.findChild(QtWidgets.QLineEdit, "searchFilter")
-    search_box.setText(TEST_STRING)
-
-    # 5) Verify if the test string is set
-    Report.result(Tests.set_search_string, search_box.text() == TEST_STRING)
-
-    # 6) Clear search string and verify if it is cleared
-    clear_text_button = search_frame.findChild(QtWidgets.QToolButton, "ClearToolButton")
-    clear_text_button.click()
-    Report.result(Tests.search_string_cleared, search_box.text() == "")
+
+    # 1) Initialize our O3DE editor object
+    qtpy_o3de_editor = QtPyO3DEEditor()
+
+    # 2) Open and initialize Script Canvas window (Tools > Script Canvas)
+    qtpy_o3de_editor.open_script_canvas()
+
+    # 3) Get a local handle on the Node Palette QtPy object
+    qtpy_node_palette = qtpy_o3de_editor.sc_editor.node_palette
```

Comment:
Move this up to step 2 and reword it a bit along the lines of "Open Script Canvas Window and initialize objects for test.

don't forget to renumber the steps here and in the desc.

Reply:
moved this up and renumber/reworded the steps